CMakeLists.txt

```diff
@@ -54,3 +54,8 @@ if (SLAC_INSTALL)
         EXPORT slac-targets
     )
 endif()
+
+if(BUILD_TESTING)
```

Comment:
Please use the if statement defined as for example here
https://github.com/EVerest/libocpp/blob/5dc9dc54b87a38814a1f503cce8c395b314b7952/CMakeLists.txt#L107
Please also add the options as here:
https://github.com/EVerest/libocpp/blob/5dc9dc54b87a38814a1f503cce8c395b314b7952/CMakeLists.txt#L17-L18

Reply:
Could you please add this two lines in the `CMakeLists.tx`
https://github.com/EVerest/libocpp/blob/5dc9dc54b87a38814a1f503cce8c395b314b7952/CMakeLists.txt#L17-L18